js/genkit/tests/embed_test.ts

```diff
@@ -30,10 +30,12 @@ describe('embed', () => {
     });
 
     it('passes string content as docs', async () => {
-      const response = await ai.embed({
-        embedder: 'echoEmbedder',
-        content: 'hi',
-      });
+      const response = (
+        await ai.embed({
+          embedder: 'echoEmbedder',
+          content: 'hi',
+        })
+      )[0].embedding;
```

Comment:
probably better to assert on the entire response... on line 45

Reply:
I only added the "[0].embedding" part and the formatter moved whitespace. I have no idea what this test is about. Whoever wrote it originally should fix it probably.